draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -565,6 +565,51 @@
 </t>
 </section>
 
+<section title="Representations" anchor="representations">
+<iref primary="true" item="representation"/>
+<iref primary="true" item="selected representation"/>
+<x:anchor-alias value="representation"/>
+<x:anchor-alias value="selected.representation"/>
+<x:anchor-alias value="selected representation"/>
+<t>
+   A <x:dfn>representation</x:dfn> is information
+   that is intended to reflect a past, current, or desired state of a given
+   resource, in a format that can be readily communicated via the protocol.
+   A representation consists of a set of representation metadata and a
+   potentially unbounded stream of representation data.
+</t>
+<t>
+   HTTP allows "information hiding" behind its uniform interface by phrasing
+   communication with respect to a transferable representation of the resource
+   state, rather than transferring the resource itself. This allows the
+   resource identified by a URI to be anything, including temporal functions
+   like "the current weather in Laguna Beach", while potentially providing
+   information that represents that resource at the time a message is
+   generated <xref target="REST"/>.
+</t>
+<t>
+   The uniform interface is similar to a window through which one can observe
+   and act upon a thing only through the communication of messages to an
+   independent actor on the other side. A shared abstraction is needed to
+   represent ("take the place of") the current or desired state of that thing
+   in our communications. When a representation is hypertext, it can provide
+   both a representation of the resource state and processing instructions
+   that help guide the recipient's future interactions.
+</t>
+<t>
+   An origin server might be provided with, or be capable of generating, multiple
+   representations that are each intended to reflect the current state of a
+   <x:ref>target resource</x:ref>. In such cases, some algorithm is used by
+   the origin server to select one of those representations as most applicable
+   to a given request, usually based on <x:ref>content negotiation</x:ref>.
+   This <x:dfn>selected representation</x:dfn> is used to provide the data
+   and metadata for evaluating conditional requests (<xref target="preconditions"/>)
+   and constructing the content for <x:ref>200 (OK)</x:ref>,
+   <x:ref>206 (Partial Content)</x:ref>, and
+   <x:ref>304 (Not Modified)</x:ref> responses to GET (<xref target="GET"/>).
+</t>
+</section>
```

Comment:
Just put anchor and irefs for "selected presentation" on the last paragraph. (or leave the subsection in place)